python/scripts/build_python_package.py

```diff
@@ -33,6 +34,10 @@ def main() -> None:
     rust_cli_cargo_toml_path = rust_root_dir / "cli" / "Cargo.toml"
     rust_cli_cargo_lock_path = rust_root_dir / "cli" / "Cargo.lock"
 
+    # Check python README links
+    python_readme_path = python_root_dir / "README.md"
+    check_readme_links(python_readme_path)
```

Comment:
let's have a more explicit name for this function, e.g., "check_readme_has_absolute_links".

Reply:
actually, I propose to make this function more generic: `check_markdown_has_only_absolute_links`. then let's remove the references about "Readme" in the body of this function, as it's now about markdown in general.

Reply:
Got it! I also updated the function to check for either directories or a file based on the file path to be more generic.